src/components/login/LoginModal.tsx

```
@@ -30,7 +30,7 @@ export default function LoginModal({ isOpen, onClose, moveToSignupModal }: Login
     e.preventDefault()
 
     apiCall("/api/user/login", "POST", { email, password }).then(({ data }: { data: any }) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)